docs/reference/globals.md

```diff
@@ -28,25 +28,62 @@ The global variables include:
 
 ## `tables`
 
-This is an object that holds all the tables for the default database (called `data`) as properties. Each of these property values is a table class that subclasses the Resource interface and provides access to the table through the Resource interface. For example, you can get a record from a table (in the default database) called 'my-table' with:
+This is an object with all the tables in the default database (the default database is "data"). Each table that has been declared or created in your `schema.graphql` file will be available as a property on this object, and the value will be the table class that can be used to interact with that table. The table classes implement the Resource API.
 
-```javascript
-import { tables } from 'harperdb';
-const { MyTable } = tables;
-async function getRecord() {
-	let record = await MyTable.get(recordId);
+**Schema Definition:**
+Tables are defined in your `schema.graphql` file using the `@table` directive. For example:
+
+```graphql
+type Product @table {
+	id: ID @primaryKey
+	name: String
+	price: Float
 }
 ```
 
-It is recommended that you [define a database](../developers/applications/defining-schemas) for all the tables that are required to exist in your application. This will ensure that the tables exist on the `tables` object. Also note that the property names follow a CamelCase convention for use in JavaScript and in the GraphQL Schemas, but these are translated to snake_case for the actual table names, and converted back to CamelCase when added to the `tables` object.
+Once declared, `Product` will be available as `tables.Product` (or `databases.data.Product`). This mapping is automatic: every table defined in the default database in your schema will appear as a property on the `tables` object. For more info, read our complete [guide on defining schemas](../developers/applications/defining-schemas).
+
+### Example
+
+```js
+const ProductTable = tables.Product; // Same as databases.data.Product
+
+// Create a new record (`id` is automatically generated when using `.create()`)
+const created = await ProductTable.create({ name: 'Shirt', price: 9.5 });
+
+// Retrieve by primary key
+const record = await ProductTable.get(created.id);
+
+// Insert or replace by ID
+await ProductTable.put(created.id, { ...record, price: 7.5 });
```

Comment:
Did you verify this works? It should work in v5, but Product table isn't defined as `loadAsInstance=false`, so I can't you can enumerate the properties like this... unless I am missing something.